### ISSUE TITLE
RChain: RChain Community in relation to lives of its members

### ISSUE THREAD

**Issue by azazime** [rchain/bounties#733]
Rchain is packed with a lot of endless possibilities and advantages for it's members and the world at large.  Hence,  understanding these advantages is very pivotal to enjoying such.  

The document below describes Rchain,  The Community and how it translates to the lives of active members. 
READ HERE: https://drive.google.com/file/d/1h6470-_6cFkRJIBlRuDUu2S-Or7PuziT/view?usp=drivesdk

**Task**
*Publish document on various social media channels✔️

**budget:** [750$]
**Measure of task**: Completed [100%]

**Comment by jasoncruzzy** [comment]
Nice work, very educating!

**Comment by Preshcode** [comment]
You did a good job there. Rchain is going to change the way the world sees blockchain technology.

**Comment by casanwugo** [comment]
@azazime. Great content! Keep that coming.

**Comment by pmoorman** [comment]
@azazime I have a hard time voting a budget for it under the Marketing label if this document is only living in Google Drive. 

I'm holding off on voting a budget till June, in anticipation of this piece getting exposure (and thus driving results)

cc @kitblake @AyAyRon-P

**Comment by pmoorman** [comment]
To put it differently: documents that never get published aren't very valuable. The value is in reaching an audience with it.

You might want to update your description, because it indicates the work is already 100% done

**Comment by azazime** [comment]
Published link of document on medium👇

https://medium.com/@andrewasamonye/rchain-as-it-relates-to-life-of-members-8be21385535e

**Comment by azazime** [comment]
Published link of document on Reddit👇

https://www.reddit.com/user/RChain_Africa/comments/8sgs4i/rchain_how_the_bounty_system_works/?utm_source=reddit-android

**Comment by azazime** [comment]
Published link of document on facebook group with over 20,000 active users👇

https://m.facebook.com/groups/438390979539823#!/groups/438390979539823?view=permalink&id=1785644351481139&refid=18&_ft_=qid.6571649239004128086%3Amf_story_key.1785644351481139%3Atop_level_post_id.1785644351481139%3Atl_objid.1785644351481139%3Asrc.22&__tn__=%2As%2As-R

**Comment by lapin7** [comment]
@azazime 
IMHO Facebook should not be used as a marketing channel.
And we should take care about sending out messages to groups, with respect being regarded as a SPAM'r.

**Comment by pmoorman** [comment]
>  IMHO Facebook should not be used as a marketing channel.

@lapin7 can you elaborate?

**Comment by lapin7** [comment]
Well, Facebook is abusing our personal data by selling it for a high price to third parties. RChain is AFAIK against this practice.

> @azazime Published link of document on facebook group with over 20,000 active users👇

> https://m.facebook.com/groups/438390979539823#!/groups/438390979539823?view=permalink&id=1785644351481139&refid=18&_ft_=qid.6571649239004128086%3Amf_story_key.1785644351481139%3Atop_level_post_id.1785644351481139%3Atl_objid.1785644351481139%3Asrc.22&__tn__=%2As%2As-R

**Comment by pmoorman** [comment]
I wasn't aware of any such policy, and I think the marketing organisation is also engaging in Facebook promotions [so if this is really the policy, then we have more places to solve it]!

Bear in mind that almost all free services on the internet also need to make money, mostly through selling data or ads. _Almost all places where we could reach exposure have this nature! (Medium, Reddit, LinkedIn, Twitter, etc.)_

Personally, I think we should separate 2 things:

1. RChain is against this practice of people not owning their own data, and is therefore building tooling to create "a more open internet" in the future

2. RChain wants to spread the word, and needs to plug into whatever places people hang out right now. Facebook happens to be such a place.

It's everyone's personal choice that if they don't want Facebook to sell their data, that they can stop using Facebook. For now, many people hang out on Facebook, and thus RChain marketing should reach there.

**TL;DR:** I don't understand why we're against using Facebook for marketing purposes, even if we dislike their revenue model. It seems highly impractical.

**Comment by lapin7** [comment]
@pmoorman
I clearly said IMHO. 
But I like the way you separate “repairing” FB and using FB
So let’s use the channels  where the folks hang out (IMHO)

**Comment by pmoorman** [comment]
oke, fair enough. Thanks for clarifying!

I personally hope someone will be able to make a better Facebook on RChain, sometime in the not-so-distant future 👍

**Comment by glenbraun** [comment]
Hi,

I’m Glen. I’m working on a compensated OSS model. (https://github.com/glenbraun/CollaborativeLicense)

Social networking is too big to be owned by one company. This model keeps the code in the open but forms contracts to compensate contributors.

**Comment by TrenchFloat** [comment]
It looks like the writers have already collected their rewards for the article before all the work was done (like getting an audience), by a mistake on the label guides' part (that includes me). That said, here is some feedback for next time:

Blanket statement concerning all articles: An article with less-than-professional quality or appeal can actually harm the image of the RChain community, producing negative value.
-Be sure to use a consistent font throughout the article
-Make sure any charts put into the document are legible. The chart on page 4 is not.
-General formatting: headers should not be separated awkwardly from their text (p1-2) or use up an entire page (p3)
-Instead of pasting a link into a Medium or Reddit page, paste the article itself.

After that, as @pmoorman said, the value of articles like these is directly related to their reach. 1 upvote on Reddit and apparently no engagement on Medium constitutes a $0 budget. The link to the Facebook page gave me "content not found."

Best practice is to have a label guide proofread before publication, but the burden is on us to reach out first. In the future, only articles with adequate representation of the community and adequate reach will receive rewards.
